pkg/kopia/command/repository.go

```diff
@@ -34,6 +34,9 @@ type RepositoryCommandArgs struct {
 	Username        string
 	RepoPathPrefix  string
 	ReadOnly        bool
+	// Only for CreateCommand
```

Comment:
what happens if someone provides this flag with non create command?

Reply:
The fields are not used in [RepositoryConnectCommand](https://github.com/kanisterio/kanister/blob/77b3c0612348988b6564d2f92318095509020359/pkg/kopia/command/repository.go#L46-L77). The same way `PITFlag` is used in Connect but not in Create